docker-compose.yml

```diff
@@ -17,6 +17,15 @@ services:
     environment:
       MINIO_ACCESS_KEY: minio
       MINIO_SECRET_KEY: minio123
+  datadag:
+    environment:
+      - DD_API_KEY
```

Comment:
Does this run without an API key set?

Reply:
The service will boot up, but you wont send logs.

Reply:
:+1: